src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -686,52 +687,62 @@ func (r *StatefulSetResource) rpkStatusContainer(
 func prepareAdditionalArguments(
 	developerMode bool,
 	originalRequests redpandav1alpha1.RedpandaResourceRequirements,
+	additionalCommandlineArguments map[string]string,
 ) []string {
 	requests := originalRequests.DeepCopy()
 
 	requestedCores := requests.RedpandaCPU().Value()
 	requestedMemory := requests.RedpandaMemory().Value()
 
-	args := []string{}
+	args := make(map[string]string)
 	if developerMode {
-		args = append(args,
-			"--overprovisioned",
-			"--kernel-page-cache=true",
-			"--default-log-level=debug",
-		)
+		args["overprovisioned"] = ""
+		args["kernel-page-cache"] = "true"
```

Comment:
Because of this change linter stated to complain:

```
pkg/resources/ingress.go:99:68: string `true` has 2 occurrences, make it a constant (goconst)
--
  | r.annotations["nginx.ingress.kubernetes.io/force-ssl-redirect"] = "true"
```

https://buildkite.com/redpanda/redpanda/builds/22710#01862dcd-a745-4e05-adec-77c7ba803d1e/408-477

Reply:
sorted